llpc/util/llpcInternal.cpp

```diff
@@ -483,14 +483,24 @@ bool IsElfBinary(
     return isElfBin;
 }
 
+// =====================================================================================================================
+// Checks whether the input data is actually LLVM bitcode
+bool IsLlvmBitcode(
+    const void* pData,    // [in] Input data to check
+    size_t      dataSize) // Size of the input data
+{
+    const unsigned char magic[] = { 'B', 'C', 0xC0, 0xDE };
+    return dataSize >= sizeof magic && memcmp(pData, magic, sizeof magic) == 0;
```

Comment:
nit: `return (dataSize >= sizeof(magic)) && (memcmp(pData, magic, sizeof(magic)) == 0)`;

Reply:
My preference is not to do that. Is there some particular coding standard that says I should?

Reply:
It doesn't seem to be in the repository yet, but look for an internal copy of `palCodingStandards.md`.
@amdrexu usually enforces that from what I've seen.

Reply:
Ah, it's in the other repo: https://github.com/GPUOpen-Drivers/pal/blob/dev/doc/process/palCodingStandards.md

Reply:
Yuck, but done :-)

Reply:
Nicolai objects to using too many () for conditions. PAL often does the opposite. We have some discussions about this. This currently depends on the developers' decision, not a must, but a recommendation. Better to add () for each condition until we finally switch to new style.

Reply:
I was mostly pointing out the lack of parens around `sizeof`'s operand, but don't care much either way.

Reply:
Sorry, I missed that, but if you don't care too much then I'll leave it as-is :-)